docs/reference/edot-sdks/nodejs/migration.md

```diff
@@ -88,18 +88,63 @@ The Elastic APM Node.js agent [`apmClientHeaders`](apm-agent-nodejs://reference/
+- To turn off metrics sending and collection overhead, use the following settings:
+    - [`OTEL_METRICS_EXPORTER=none`](https://opentelemetry.io/docs/specs/otel/configuration/sdk-environment-variables/#exporter-selection) to turn off the sending of any metrics.
+    - [`ELASTIC_OTEL_HOST_METRICS_DISABLED=true`](/reference/edot-sdks/nodejs/configuration.md#elastic_otel_host_metrics_disabled-details) to remove the overhead from metrics collection by the `@opentelemetry/host-metrics` package.
+    - [`OTEL_NODE_DISABLED_INSTRUMENTATIONS=runtime-node`](/reference/edot-sdks/nodejs/configuration.md#otel_node_disabledenabled_instrumentations-details) to remove metrics collection overhead from the `@opentelemetry/instrumentation-runtime-node` instrumentation
+
+For example:
+
+```
+ELASTIC_OTEL_CONTEXT_PROPAGATION_ONLY=true
+OTEL_METRICS_EXPORTER=none
+ELASTIC_OTEL_HOST_METRICS_DISABLED=true
+OTEL_NODE_DISABLED_INSTRUMENTATIONS=runtime-node
+```
+
 ### `disableInstrumentations`
 
 The Elastic APM Node.js agent [`disableInstrumentations`](apm-agent-nodejs://reference/configuration.md#disable-instrumentations) option corresponds to the EDOT Node.js [`OTEL_NODE_DISABLED_INSTRUMENTATIONS`](/reference/edot-sdks/nodejs/configuration.md#otel_node_disabledenabled_instrumentations-details) option.
 
 For example: `OTEL_NODE_DISABLED_INSTRUMENTATIONS=express,mysql`.
 
+### `disableMetrics`
+
+The Elastic APM Node.js agent [`disableMetrics`](apm-agent-nodejs://reference/configuration.md#disable-metrics) option corresponds to the EDOT Node.js [`OTEL_METRICS_EXPORTER`](https://opentelemetry.io/docs/specs/otel/configuration/sdk-environment-variables/#exporter-selection) option. Set it to none to disable the collection of metrics.
+
+For example: `OTEL_METRICS_EXPORTER=none`.
+
+### `disableSend`
+
+The Elastic APM Node.js agent [`disableSend`](apm-agent-nodejs://reference/configuration.md#disable-send) does not corresponds to any Opentelemetry option. You can get the same behavior by setting to the value `none` on the following environment vars dedicated to [exporter selection](https://opentelemetry.io/docs/specs/otel/configuration/sdk-environment-variables/#exporter-selection). Note that you have control per signal.
+
+- `OTEL_TRACES_EXPORTER` for traces.
+- `OTEL_METRICS_EXPORTER` for metrics.
+- `OTEL_LOGS_EXPORTER` for logs.
+
+For example `OTEL_TRACES_EXPORTER=none OTEL_METRICS_EXPORTER=none OTEL_LOGS_EXPORTER=none` will completelly disable sending but `OTEL_TRACES_EXPORTER=none` will disable only the sending of traces.
+
 ### `environment`
 
 The Elastic APM Node.js agent [`environment`](apm-agent-nodejs://reference/configuration.md#environment) option corresponds to setting the `deployment.environment` key in [OTEL_RESOURCE_ATTRIBUTES](https://opentelemetry.io/docs/concepts/sdk-configuration/general-sdk-configuration/#otel_resource_attributes).
```

Comment:
nit: Can we update this to the current `deployment.environment.name`?  `deployment.environment` was deprecated relatively recently.  This could be done in a separate change.